modules/changinSection.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable semi */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
```

Comment:
- [ ] Same here, Please remove disable lines for `eslint` and Fix all linters issues.